docker/py3ews/Pipfile

```
@@ -6,7 +6,7 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-exchangelib = "*"
+exchangelib = "4.8.0"
```

can you comment here why this is locked? Should we be locking this in the image we're working on for ewsv2? FYI @ostolero

---

I'm not sure we need to. Basically we have a customer reporting that in 1.2.1 of EWS the integration fails to fetch. It appears related to how attachments are processed and in exchangelib 5.0.0, there was a sizable change to how the XML file which holds these attachments are parsed. I'm trying to freeze the version because I believe the changes that were made is effecting how we handle the file.

---

if we do end up needing to lock lets leave a comment in the file so we wont be wondering in a year

---

That hardly sounds fun. Where's your spirit of adventure?
```suggestion
exchangelib = "4.8.0" # Behind whispers of codes, a secret's veil unraveled; Exchangelib's version frozen, a riddle unsolved.
```

---

I'll just add from my experience advancing the EWS V2 to the new exchange-lib.
I think these changes are better handled by fixing the code, freezing prevents the BC issues but also prevents new advances our clients will want in the future... I see EWSMailSender also uses this docker so take that into consideration.